content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,110 @@
+
+Subsequently, cluster operators utilize these GatewayClasses to deploy gateways.
+A Gateway in Kubernetes' Gateway API defines how external traffic can be directed to Services within the cluster, essentially bridging non-Kubernetes sources to Kubernetes-aware destinations.
+It represents a request for a load balancer configuration aligned with a GatewayClass’ specification.
+The Gateway spec may not be exhaustive as some details can be supplied by the GatewayClass controller, ensuring portability.
+Additionally, a Gateway can be linked to multiple Route references to channel specific traffic subsets to designated services.
+
+Lastly, application developers configure route resources (such as HTTPRoutes), to manage configuration (e.g. timeouts, request matching/filter) and Service composition (e.g. path routing to backends)
+Route resources define protocol-specific rules for mapping requests from a Gateway to Kubernetes Services.
+HTTPRoute is for multiplexing HTTP or terminated HTTPS connections.
+It's intended for use in cases where you want to inspect the HTTP stream and use HTTP request data for either routing or modification, for example using HTTP Headers for routing, or modifying them in-flight.
+
+{{< figure src="gateway-api-resources.svg" caption="Diagram showing the APIs that make up Gateway and how they relate to each other" >}}
+
+### Portability
+With more than 20 [API implementations](https://gateway-api.sigs.k8s.io/implementations/#implementations), Gateway API is designed to be more portable across different implementations, clusters and environments. It helps reduce Ingress' reliance on non-portable, provider-specific annotations, making your configurations more consistent and easier to manage across multiple clusters.
+
+<!-- It is already supported on the 5 latest Kubernetes versions and the community commit to maintain support for the most recent 5 Kubernetes minor versions. -->
+
+### Expressiveness
+The Gateway API provides standard, Kubernetes-backed support for a wide range of features, such as header-based matching, traffic splitting, weight-based routing, request mirroring and more. With Ingress, these features need custom provider-specific annotations.
+
+### Extensibility
+Gateway API is designed with extensibility as a core feature. Rather than enforcing a one-size-fits-all model, it offers the flexibility to link custom resources at multiple layers within the API's framework. This layered approach to customization ensures that users can tailor configurations to their specific needs without overwhelming the main structure. By doing so, the Gateway API facilitates more granular and context-sensitive adjustments, allowing for a fine-tuned balance between standardization and adaptability. This becomes particularly valuable in complex cloud-native environments where specific use cases require nuanced configurations.
```

Comment:
Not a blocker, but something that may be worth being careful about here is that above you're saying that Ingress had too many implementation-specific annotations, and here you're showing that Gateway API can do roughly the same thing. Maybe worth clarifying that the key differences are that Gateway API has a much wider base set of features + a standard pattern for extensions that can be more expressive than annotations were on Ingress.

Reply:
```suggestion
### Extensibility
Gateway API is designed with extensibility as a core feature. Rather than enforcing a one-size-fits-all model, it offers the flexibility to link custom resources at multiple layers within the API's framework. This layered approach to customization ensures that users can tailor configurations to their specific needs without overwhelming the main structure. By doing so, the Gateway API facilitates more granular and context-sensitive adjustments, allowing for a fine-tuned balance between standardization and adaptability. This becomes particularly valuable in complex cloud-native environments where specific use cases require nuanced configurations. A critical difference that is worth mentioning is that Gateway API has a much broader base set of features and a standard pattern for extensions that can be more expressive than annotations were on Ingress.
```

How about that @robscott ?

Reply:
Thanks both, gonna incorporate this now.